web/docs/database/backends.md

```diff
@@ -1,3 +1,182 @@
 ---
-title: Database Backends
```

Comment:
I would add at the end an `Options Reference` section with:
- `db`
- `db.system`
- `db.seeds`
- `db.prisma`
fields explained. 

When you do that, you'll see that you can delete the duplicate explanations above and keep the document simpler.